### PR TITLE
add create and delete review functionality

### PR DESCRIPTION
**Pull request by Hajny25**

### CODE REVIEW

File: src/repositories/review_repository.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		review.Comment,
+		review.Datetime,
+		review.IsSpoiler != nil && *review.IsSpoiler,
+		utils.MysqlUuid(review.UserID),
+		utils.MysqlUuid(review.MovieID),
+	)
+
+	_, err := stmt.Exec(rr.DatabaseManager.GetDatabaseConnection())
+	if err != nil {
+		fmt.Println(err)
```
Comment (Kieberts) [inline src/repositories/review_repository.go:45]: remove or change to log. Also why do you not control if rows were affected?

File: src/repositories/review_repository.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		return kts_errors.KTS_INTERNAL_ERROR
+	}
+	return nil
+}
+
+func (rr *ReviewRepository) DeleteReview(id *uuid.UUID) *models.KTSError {
+	stmt := table.Reviews.DELETE().WHERE(table.Reviews.ID.EQ(utils.MysqlUuid(id)))
+	_, err := stmt.Exec(rr.DatabaseManager.GetDatabaseConnection())
```
Comment (Kieberts) [inline src/repositories/review_repository.go:53]: Same here. There might not be an error while executing, but if nothing was actually delete, it should return an error.

File: src/controllers/review_controller_test.go [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+			reviewController := ReviewController{
+				ReviewRepo: reviewRepoMock,
+			}
+
+			// define expectations
+			tc.setExpectations(*reviewRepoMock, tc.reviewData)
+
+			// WHEN
+			// call CreateReview with review data
+			_, err := reviewController.CreateReview(tc.reviewData)
```
Comment (Kieberts) [inline src/controllers/review_controller_test.go:104]: check result.  Also test return value for Create Review

File: src/controllers/review_controller_test.go [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+			err := reviewController.DeleteReview(&tc.id)
+
+			// THEN
+			// check expected error and id
+			assert.Equal(t, err, tc.expectedError, "wrong error")
+		})
+	}
+}
+
+func getSampleReviewRequest() models.CreateReviewRequest {
```
Comment (Kieberts) [inline src/controllers/review_controller_test.go:162]: Move to own file in utils

File: src/handlers/review_handler_test.go [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+					Comment:   body["comment"].(string),
+					Datetime:  datetime,
+					IsSpoiler: body["isSpoiler"].(bool),
+					UserID:    body["userId"].(string),
+					MovieID:   body["movieId"].(string),
+				}
+				id := uuid.New()
+				mockCtrl.EXPECT().CreateReview(reviewData).Return(&id, nil)
+			},
+			expectedStatus: 200,
```
Comment (Kieberts) [inline src/handlers/review_handler_test.go:50]: Should Successful Creation not be 201?

**Review by Kieberts — APPROVED** [review]

File: src/controllers/review_controller_test.go (hunk 3 above)
Comment (Kieberts) [reply src/controllers/review_controller_test.go:104]: .

**Review by Kieberts — CHANGES_REQUESTED** [review]

File: src/controllers/review_controller_test.go (hunk 4 above)
Comment (Hajny25) [reply src/controllers/review_controller_test.go:162]: we agreed on moving everything to a samples package, but I'm not going to bother with that either, very low priority

**Review by Kieberts — APPROVED** [review]
Thank you and good job

**Pull request merged**